packages/talos_agent/src/agent/state_manager.rs

```diff
@@ -180,6 +180,16 @@ impl<TSignalTx: Sender<Data = Signal> + 'static> StateManager<TSignalTx> {
                 decision: message.decision.clone(),
                 version: message.version,
                 safepoint: message.safepoint,
+                conflict: match message.conflicts {
```

Comment:
With the last PR that I merged yesterday, the certifier will only pass the conflict version under a property called `conflict_version`. Could you sync that change here?